### ISSUE TITLE
Conflicting VoiceOver results for test 3 "Request information about a menu button" (Navigation Menu Button, V24.10.12)

### ISSUE THREAD

**Issue by IsaDC** [w3c/aria-at#1136]
## Description of Behavior

This issue also occurs in **Test 6: Request information about a menu item**. 

Tester `tactics2` recorded the output "No output was detected" for all assertions, whereas both the bot and I recorded the output: 

"WAI-ARIA Quick Links  
menu pop-up button  
is in the VoiceOver cursor/has keyboard focus."

## Test Setup

- Test File: [test-09-reqInfoAboutMenuButton-voiceover_macos.collected.html](https://aria-at.netlify.app/tests/menu-button-navigation/test-09-reqInfoAboutMenuButton-voiceover_macos.collected.html)
- AT: VoiceOver for macOS (Version 11.6 and above)
- Browser: Safari

## Review Conflicts for "Request information about a menu button"

1. ### Assertion Results for "Control+Option+F3" Command and "State 'collapsed' is conveyed" Assertion

* Tester IsaDC recorded output "WAI-ARIA Quick Links
              menu pop up             button             is in the VoiceOver cursor                           " and marked assertion as failing.
* Tester tactics2 recorded output "No output was detected." and marked assertion as failing.

2. ### Assertion Results for "Control+Option+F3" Command and "State 'collapsed' is conveyed" Assertion

* Tester IsaDC recorded output "WAI-ARIA Quick Links
              menu pop up             button             is in the VoiceOver cursor                           " and marked assertion as failing.
* Tester tactics2 recorded output "No output was detected." and marked assertion as failing.

3. ### Assertion Results for "Control+Option+F4" Command and "State 'collapsed' is conveyed" Assertion

* Tester IsaDC recorded output "WAI-ARIA Quick Links
              menu pop up             button             has keyboard focus              " and marked assertion as failing.
* Tester tactics2 recorded output "No output was detected." and marked assertion as failing.

4. ### Assertion Results for "Control+Option+F4" Command and "State 'collapsed' is conveyed" Assertion

* Tester IsaDC recorded output "WAI-ARIA Quick Links
              menu pop up             button             has keyboard focus              " and marked assertion as failing.
* Tester tactics2 recorded output "No output was detected." and marked assertion as failing.

**Comment by IsaDC** [comment]
Closing this issue because the lack of output was due to the function key setting on an external keyboard.
